assembl/static2/js/app/components/common/navbar.jsx

```diff
@@ -21,7 +20,9 @@ class NavBar extends React.Component {
     });
   }
   componentWillReceiveProps() {
-    this.setState({ isMenuHidden: true });
+    this.state = {
+      isMenuHidden: true
+    };
```

Comment:
You should only use 'this.state ='  in the constructor, everywhere else use setState.